server/protx/data/api/urls.py

```diff
@@ -3,10 +3,19 @@
 from protx.data.api import views
 
 app_name = 'protx_api'
+
+
 urlpatterns = [
+    url('analytics/', views.get_analytics, name='data'),
+    path('analytics-plot-distribution/<area>/<geoid>/<variable>/<unit>/', views.get_analytics_distribution_plot_data, name='data'),
+
+    url('demographics/', views.get_demographics, name='data'),
+    path('demographics-plot-distribution/<area>/<geoid>/<variable>/<unit>/', views.get_demographics_distribution_plot_data, name='data'),
+
     url('display/', views.get_display, name='data'),
+
     url('maltreatment/', views.get_maltreatment, name='data'),
-    url('demographics/', views.get_demographics, name='data'),
-    url('resources/', views.get_resources, name='data'),
-    path('demographics-plot-distribution/<area>/<geoid>/<variable>/<unit>/', views.get_demographics_distribution_plot_data, name='data')
```

Comment:
demographics-plot-distribution route was originally returning historgram/distrubution data.  Then in https://github.com/TACC/protx/pull/83/commits/8a2cb73e0e446a572b2daa2ca113cdf0c5c90cf9 in https://github.com/TACC/protx/pull/83, we started using the existing that route ( `demographics-plot-distribution) to return the actual plot configuration/description.  

We should probably rename it now to something more appropriate.  `demographics-plot`?

Reply:
The route naming was bugging me but I left it be because I thought it maybe made sense form the back end perspective. Now that I have better grokked the end to end, I fully agree. Same for `maltreatment` (and eventually analytics, etc.).

Reply:
👍   

`maltreatment` is okay but 👍  for updating `maltreatment-plot-distribution` route too.